fix(init): fixes error related to changed API where label(...) is now label(text=...)

alma3lol opened the pull request.

alma3lol committed:
fix(init): fixes error related to changed API where label(...) is now label(text=...)

icecreammatt reviewed (approved):
I was going to submit a PR to make this same change. Glad I saw this one first. Hopefully it can get merged in.

Without this change when going to the text panel and selecting the plugin this error shows up:

```python
Traceback (most recent call last):
  File "/Users/matt/Library/Application Support/Blender/3.5/scripts/addons/bpy_externall/__init__.py", line 184, in draw
    col.label('listening on ' + statemachine['tempfile'])
TypeError: UILayout.label(): required parameter "text" to be a keyword argument!
```

Once the change is made that error goes away and then toggling the switch fixes the plugin so you don't have to enable it in the Blender console.

alma3lol commented:
@zeffii Please merge this pull request

alma3lol committed:
fix: fixes missing context.override from the new API

the new API added a context.override call so bpy.ops.text.run_script is called without context param.
this commit adds call to context.override to resolve this
